tests/update/time-vector.md

```diff
@@ -3,11 +3,11 @@
 ```sql,creation
 CREATE TABLE time_vector_data(time TIMESTAMPTZ, value DOUBLE PRECISION);
 INSERT INTO time_vector_data VALUES
-    ('2020-1-1', 30.0),
-    ('2020-1-2', 45.0),
-    ('2020-1-3', NULL),
-    ('2020-1-4', 55.5),
-    ('2020-1-5', 10.0);
+    ('2020-1-1 UTC', 30.0),
```

Comment:
Were you seeing this break with the new change?  I would have thought the timezone on the insert would just default to the database setting.

Reply:
Tests pass locally for me still with this change for both this file and the time weighted average tests.